packages/go/test/fixtures/24-bad-handler/vercel.json

```diff
@@ -0,0 +1,9 @@
+{
+  "version": 2,
+  "probes": [
+    {
+      "path": "/api/bad-receiver",
+      "mustContain": "version:go1.13.15:from GoodHandler"
+    }
```

Comment:
I ran this test without the changes to make sure it fails. It does: https://github.com/vercel/vercel/runs/7911138926?check_suite_focus=true

Reply:
Lets rename this `vercel.json` file to `probes.json` and remove `version`

Reply:
Sure thing: [`550fd0f` (#8422)](https://github.com/vercel/vercel/pull/8422/commits/550fd0f8582f26910834a4654caef704c6a46ae6)